server/routes/index.js

```diff
@@ -2,9 +2,16 @@ import { Router } from 'express';
 
 import userRouter from './users';
 import articleRouter from './article';
+import { goodHttpResponse } from '../utilities/httpResponse';
+
 
 const router = Router();
 
+router.get('/', (request, response) => {
```

Comment:
@ProfJigsaw please why did you add this here as well?

Reply:
We need to handle the get request to the first version of the app.

Reply:
Not sure I follow, is this with regards to api versioning?

Reply:
Yes it is

Reply:
we don't need to do this yet. please remove

Reply:
Removed